docs/docs/jaseci_architype/take.md

```diff
@@ -0,0 +1,264 @@
+## Skipping and Disengaging
+
+Jac offers couple of more useful control statements that are pretty convenient, `skip` and `disengage`, with walker traversing graphs with `take` commands.
+
+### Skipping
+
+The idea behind the abstraction of `skip` in the context of a walkers code block is that it tells a walker to halt and abandon any unfinished work on the current node in favor of moving to the next node (or complete computation if no nodes are queued up).
+
+> **Note**
+>
+> Node/edge abilities also support the usage of the skip directive. The skip merely decides not to use the remaining steps of that `ability` itself in this context.
+
+
+Lets change the `init` walker of **Example 3** to demostrate how the `skip` command works;
```

Comment:
@RolexAlexander Does it make sense to be indexing the examples in the docs like this? `Example 3`. Is there an example 3 in the docs actually? If so, we should make the `Example 3` text a hyper link to it. If not, we should remove it. There are several places here.